examples/step-70/doc/intro.dox

```diff
@@ -426,27 +426,51 @@ of processor and the number of cells in $\Gamma$ increases, this leads
 to a severe bottleneck in terms of memory. Consequently, an alternative strategy is sought
 in this step.
 
-When looking at the formulation for both the penalization approach ($\mathcal{L}^2$ or $\mathcal{H}^1$)
-and the Nitsche method, we can come to the conclusion that we only require limited information
-related to $\Gamma$ on its quadrature points that is:
-- Their reference location with regard to the embedding fluid cell;
-- The weights times the Jacobian of the transformation, i.e. `JxW`.
 
+<h4>Representation of Ω and Γ</h4>
+
+Remember that for both the penalization approach ($\mathcal{L}^2$ or $\mathcal{H}^1$)
+and the Nitsche method, we want to compute integrals that are approximated by
+the quadrature. That is, we need to compute
+\f[
+\beta (\textbf{v},\textbf{u})_{\Gamma} =
+\sum_{K\in \Gamma} \sum_{i=1}^{n_q}  \big(\hat{\textbf{u}}(\hat x_i)  (\textbf{v} \circ F_{K}) (\hat x_i) J_K (\hat x_i) w_i \big)
+\f]
+If you followed the discussion above, the you will recall that $\textbf{u}$
```

Comment:
```suggestion
If you followed the discussion above, you will recall that $\textbf{u}$
```

Reply:
I meant to say "then you will recall". Fixed.